Jint.Tests.CommonScripts/ConcurrencyTest.cs

```diff
@@ -1,4 +1,4 @@
-using Esprima;
+using Acornima;
```

Comment:
Would have been easier to just rename Acornina to Esprima ;)

Reply:
Definitely, just the name wouldn't really reflect the content then... 😄